src/server/sql/user/insert_new_user.sql

```diff
@@ -2,4 +2,6 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-INSERT INTO users(email, password_hash) VALUES (${email}, ${passwordHash})
+INSERT INTO users(email, password_hash)
+VALUES (${email}, ${passwordHash})
+ON CONFLICT (email) DO UPDATE SET email=${email};
```

Comment:
I'm not sure I understand what this is doing. I assumed that it changed their password, but that does not seem to be the case. Is this saying that if a user with that email already exists, then update their email? That doesn't seem sane.

Reply:
Also, this should abort on a collision, the query to insert a new user should fail if the new user already exists.

Reply:
Yeah, this is not correct. I'll change it in the next commit.